lib/house.rb

```diff
@@ -0,0 +1,50 @@
+class House
+  attr_reader :address, :rooms
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @market_average = 500000
+  end
+
+  def price
+    (@price.delete("$")).to_i
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    return true if price > @market_average
+      false
```

Comment:
i like this guard clause, although you don't need these explicit true/falses here. 

```
def above_market_average?
   price > @market_average
end
```

Since `price > @market_average` is a comparison, it already returns the correct boolean value as a result, so that's really all you need here.

Reply:
I'm really bad about that and do it all the time... It's just the way I think I guess.  I'll try to work on it.

Reply:
it's not a big deal; just something to have floating around in your head 🙂